quickstart/dev/Tiltfile

```diff
@@ -447,7 +447,6 @@ local_resource(
     allow_parallel = True,
     auto_init = run_apps,
     resource_deps = [
```

Comment:
Please don’t update anything under QuickStart. It gets auto bumped In concourse

Reply:
still a no-op? the same diff also applied on the "principal"/source